src/scf-release/packages/suse-fuse/spec

```diff
@@ -0,0 +1,3 @@
+---
+name: suse-fuse
+files:
```

Comment:
I would prefer

```
files: []
```
to indicate that this is an empty list and not just `nil`, but I guess it doesn't make any difference. However, it also makes it obvious that there are not missing lines at the end that got accidentally deleted...

Reply:
Done with commit 795b58f9da96e0e6342529f0c244d37648fd4a6d